test/ussd_clinic.test.js

```diff
@@ -507,15 +478,17 @@ describe("app", function() {
         });
 
         describe("dialback sms testing", function() {
+            var setupInputs = [
+                {session_event: 'new'}  // dial in
+                , "1"  // state_start - yes
+                , "1"  // state_consent - yes
+                , {session_event: 'close'}
+            ];
+
             it("send if redial sms not yet sent (identity loads without redial_sms_sent defined)", function() {
                 return tester
                 .setup.user.addr("27820001001")
-                .inputs(
-                    {session_event: 'new'}  // dial in
-                    , "1"  // state_start - yes
-                    , "1"  // state_consent - yes
-                    , {session_event: 'close'}
-                )
+                .inputs.apply(this, JSON.parse(JSON.stringify(setupInputs)))
```

Comment:
Ah sorry, I meant to explain this in the commit message, thanks for catching it. The problem is that in this test the argument that's passed to `inputs()` is modified. My JavaScript knowledge is a little rusty but I guess it's to do with [pass-by-reference/pass-by-value](https://stackoverflow.com/questions/518000/is-javascript-a-pass-by-reference-or-pass-by-value-language).

So the inputs is modified to be:

```
[ { session_event: 'new',
    from_addr: '27820001001',
    message_id: '823370d2-be74-4f6c-3c11-8f437f9ff27a',
    content: null,
    helper_metadata: {} },
  '1',
  '1',
  { session_event: 'close',
    from_addr: '27820001001',
    message_id: '9efec2ff-6701-40dc-8f12-aab043543ece',
    content: null,
    helper_metadata: {} } ]
```

Which causes the following tests to fail. We can get around that by turning into JSON and back into JavaScript before using it.